kore/src/Log.hs

```diff
@@ -228,8 +233,11 @@ instance MonadTrans LoggerT where
 
 logWith
     :: Entry entry
-    => LogAction m SomeEntry
+    => LogAction m ActualEntry
     -> entry
     -> m ()
 logWith logger entry =
-    logger Colog.<& toEntry entry
+    fromLogAction logger Colog.<& toEntry entry
+
+fromLogAction :: From b a => LogAction m a -> LogAction m b
```

Comment:
Please give an explicit quantifier,

```suggestion
fromLogAction :: forall a b. From b a => LogAction m a -> LogAction m b
```

and use type application on at least one of the quantified type variables everywhere that `fromLogAction` is used.